ludwig/encoders/binary_encoders.py

```diff
@@ -48,9 +48,10 @@ def __init__(
         super().__init__()
         logger.debug(' {}'.format(self.name))
 
-    def call(self, inputs, training=None, mask=None):
+    def forward(self, inputs, training=None, mask=None):
```

Comment:
Could you remove the `training` and `mask` arguments? I believe we aren't using them anymore in torch.

Reply:
removed